### DISCUSSION TITLE
Send mod+A shortcut as keystrokes to application

### DISCUSSION THREAD

**Discussion by RishavT** [i3/i3#4894] [Q&A] [answered]
I use Zoom for meetings - and it uses Alt+A to mute / unmute. I use Alt as my modifier key.

When I press Alt+A, it doesn't mute/unmute on zoom (which is expected as i3 intercepts it as some command).

Is there any way to specify in the i3 config - that when I press Alt+A, it should send the entire key press - instead of treating it as a special key combination?

Thanks!

**Comment by Airblader** [comment]
There's nothing in i3 for this, but you can probably do it using `xdotool` (and then bind that command from i3).

**Reply by RishavT** [reply]
oh, mod+a was mapped to something. I removed that and it worked. My bad. Thanks a lot!

**Reply by Airblader** [reply]
I figured you meant doing it from anywhere and not just when Zoom is focused, but fair enough, glad it worked!

**Reply by RishavT** [reply]
Yea, it was a stupid error on my part lol.

**Comment by RishavT** [comment, marked as answer]
My config had mod+a bound to something, removed that and it worked.
